hw/chisel/src/main/scala/snax/streamer/DataMover.scala

```diff
@@ -219,6 +219,12 @@ class DataMover(
   data_movement_done := (io.addr_gen_done && tcdm_req_ready) || io.zeroLoopBoundCase
   io.data_movement_done := data_movement_done
 
+  // TcdmReq is modified to include the strobe signal, so it should be assigned
```

Comment:
strobe?

Reply:
The byte strobe is a signal that's part of the TCDM already. The PULP people have it. So it's masking but on the byte level. For example, say the data width is 64 bits wide, the strobe size is 8 bits wide. If strobe bit 0 is high, then the first byte of the 64-bit data will be written to the memory. Otherwise, it will mask it.

Reply:
Aha okay, weird. I guess it makes sense to keep the same name as TCDM in that case!